### PR TITLE
UX fixes

### PR DESCRIPTION
**Pull request by mbektas**
Addressing UX issues including some reported in https://github.com/jupyterlab/jupyterlab-desktop/issues/477

- [x] Fix handling of copy - cut - paste - undo -redo keyboard shortcuts in native popup dialogs
- [x] Fix handling of `Esc` key clicks on popup dialogs
- [x] Close application when window is closed
- [x] Remove spellcheck from python path input
- [x] Disable refreshing embedded browser
- [x] Hide unused menu items on macOS app menu
- [x] Patch for index.html shown as app window title
- [x] Add option to disable auto-updates on macOS
auto-updates-disabled
auto-updates-enabled

### CODE REVIEW

File: src/main/shortcuts.ts [hunk 1]
```diff
@@ -162,9 +160,9 @@ class KeyboardShortcutManager implements IShortcutManager {
    * The enabled shortcuts
    */
   private _shortcuts: IKeyboardShortcut[] = [
-    { accelerator: 'CmdOrCtrl+c', command: this.copy.bind(this) },
-    { accelerator: 'CmdOrCtrl+v', command: this.paste.bind(this) },
-    { accelerator: 'CmdOrCtrl+x', command: this.cut.bind(this) },
+    { accelerator: 'CmdOrCtrl+R', command: this.reload.bind(this) },
```
[the remark this answers is not in the document]
Comment (mbektas) [reply src/main/shortcuts.ts:163]: good point. I will try to find an alternative solution.

[the remark this answers is not in the document]
Comment (mbektas) [reply src/main/shortcuts.ts]: I pushed a change to fix this issue.

File: src/main/shortcuts.ts [hunk 2]
```diff
@@ -88,8 +76,15 @@ class KeyboardShortcutManager implements IShortcutManager {
     this._window.setMenuBarVisibility(wasMenuBarVisible);
   }
 
+  reload() {
+    // page reload is not allowed
+  }
+
```
Comment (krassowski) [inline src/main/shortcuts.ts:82]:
```suggestion
```

Not in use anymore?

Comment (mbektas) [reply src/main/shortcuts.ts:82]: correct. good catch!

**Review by krassowski — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**